sql/02-create-table-friend.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE student_info.friend(
+  id1 INT REFERENCES student_info.student(id),
+  id2 INT REFERENCES student_info.student(id)
+);
```

Comment:
good work creating an intersection table 👍  Again this could be called simply student, however this does match with how you named your student table so good job